metagraph-plugin/conda_build_dir/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "metagraph-katana" %}
+{% set version = "0.0.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://files.pythonhosted.org/packages/b2/cf/8faf3ae5b3b107513f8c497462a2f63a9a93ba1c7efbbd86752e5138d80c/metagraph-katana-0.0.5.tar.gz
+  sha256: 27f0d316bf42af1cf400f0dd2cf650cce8b123460982e5796c015ccf065aceab
```

Comment:
I would much rather have this build from the current source directory like we have in the other packages: https://github.com/KatanaGraph/katana/blob/2bbb8e06c4b2f84ff3f1712fb470747194692794/conda_recipe/meta.yaml#L6. The reason is that it allows building the package from the source tree directly which makes a lot of sense for conda packages like this one where the packaging is in the same repo as the source. The remote download approach is best if you are packaging external code and packaging repo is separate from the source repo.